### PR TITLE
[Concept Entry] Sklearn gaussian processes

### PR DESCRIPTION
**Pull request by Radhika-okhade**
### Description

A new concept entry on the `Gaussian Processes` concept under Sklearn

### Issue Solved

#4896

### Type of Change

- Adding a new entry

### Checklist

- [/ ] All writings are my own.
- [ /] My entry follows the Codecademy Docs style guide.
- [ /] My changes generate no new warnings.
- [ /] I have performed a self-review of my own writing and code.
- [ /] I have checked my entry and corrected any misspellings.
- [ /] I have made corresponding changes to the documentation if needed.
- [/ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ /] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ /] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/sklearn/concepts/gaussian-processes/gaussian-processes.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+from sklearn.gaussian_process.kernels import RBF
+
+# Define the Gaussian Process Regressor
+gp_regressor = GaussianProcessRegressor(kernel=RBF(), alpha=1e-10)
+
+# Fit the model on the training data (X) and labels (y)
+gp_regressor.fit(X, y)
+
+# Predict the mean and standard deviation for new data points
+y_pred, sigma = gp_regressor.predict(X_test, return_std=True)
```
Comment (PragatiVerma18) [inline content/sklearn/concepts/gaussian-processes/gaussian-processes.md:36]: This is the entire code for predicting the mean and standard deviation. We should only include the part where the Gaussian Process Regressor is defined and explain the parameters.

Refer official docs [here](https://scikit-learn.org/dev/modules/generated/sklearn.gaussian_process.GaussianProcessRegressor.html).

File: content/sklearn/concepts/gaussian-processes/gaussian-processes.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+from sklearn.gaussian_process import GaussianProcessClassifier
+
+# Define the Gaussian Process Classifier
+gp_classifier = GaussianProcessClassifier(kernel=None, n_restarts_optimizer=10)
+
+# Fit the model on the training data (X) and labels (y)
+gp_classifier.fit(X, y)
+
+# Predict class probabilities for new data points
+y_prob = gp_classifier.predict_proba(X_test)
```
Comment (PragatiVerma18) [inline content/sklearn/concepts/gaussian-processes/gaussian-processes.md:51]: This is the entire code for predicting the class probabilities. We should just include the part where the Gaussian Process Classifier is defined and explain the parameters.

Refer official docs [here](https://scikit-learn.org/1.5/modules/generated/sklearn.gaussian_process.GaussianProcessClassifier.html).

**Review by PragatiVerma18 — CHANGES_REQUESTED** [review]
Hey @Radhika-okhade, thanks for adding the new entry. I have added some suggestions, please check them.

This entry seems a little difficult for beginners, please do the changes accordingly.

File: content/sklearn/concepts/gaussian-processes/gaussian-processes.md (hunk 1 above)
Comment (Radhika-okhade) [reply content/sklearn/concepts/gaussian-processes/gaussian-processes.md]: done

File: content/sklearn/concepts/gaussian-processes/gaussian-processes.md (hunk 2 above)
Comment (Radhika-okhade) [reply content/sklearn/concepts/gaussian-processes/gaussian-processes.md]: done

**Comment by Radhika-okhade** [comment]
Hey! @PragatiVerma18 I have made the necessary changes.

**Review by PragatiVerma18 — APPROVED** [review]
Hey @Radhika-okhade, thanks for addressing the comments, this lookd good to go! 🚀

**Pull request merged**